arch/x86/Kconfig

```diff
@@ -48,6 +48,85 @@ config X86_64
 	select SCHED_IPI_SUPPORTED
 	select X86_MMU
 
+menu "x86 Features"
+
+if FPU || X86_64
+
+config X86_MMX
+	bool "Enable MMX Support"
+	depends on X86
+	help
+	  This option enables MMX support, and the use of MMX registers
+	  by threads.
+
+config X86_SSE
```

Comment:
Not quite sure about this ontology. We introduce X86_SSE, but there is still CONFIG_SSE elsewhere. SSE is x86-specific.